src/lib.rs

```diff
@@ -58,6 +58,9 @@
 //! | `aws-auth`          | Enable support for the MONGODB-AWS authentication mechanism.                                                                          | `reqwest` 0.11                      | no      |
 //! | `bson-uuid-0_8`     | Enable support for v0.8 of the [`uuid`](docs.rs/uuid/0.8) crate in the public API of the re-exported `bson` crate.                    | n/a                                 | no      |
 //! | `bson-chrono-0_4`   | Enable support for v0.4 of the [`chrono`](docs.rs/chrono/0.4) crate in the public API of the re-exported `bson` crate.                | n/a                                 | no      |
+//! | `zlib-compression`  | Enable support for compressing messages with zlib                                                                                     | `flate2`                            | no      |
```

Comment:
can we linkify `zlib`, `zstd`, and `snappy`? Linking to their homepages rather than the rust crates would be good.

```suggestion
//! | `zlib-compression`  | Enable support for compressing messages with zlib.                                                                                     | `flate2`                            | no      |
```